LICENSE_header.txt

```diff
@@ -0,0 +1,25 @@
+/**
+ * disq-bio
+ *
+ * MIT License
+ *
+ * Copyright (c) 2018 disq-bio contributors
```

Comment:
Same as before.

Reply:
I was going by the suggestions in #8.

Reply:
I thought that we said "[project name] AUTHORS", and the project is DISQ in the DISTQ-BIO organization. Otherwise, members of the whole organization holds the copyright...